tests/testthat/test-length.R

```diff
@@ -1,15 +1,214 @@
-suppressWarnings({
-  library(haven)
-  library(readxl)
+#' Test `xportr_length()` function
+#'
+#' Tests will check for:
+#'  * Errors
+#' * Result of call will create `SASlength` attribute (`width` for each
+#' variable)
+
+
+test_that("xportr_length: Accepts valid domain names in metadata/metacore", {
```

Comment:
```suggestion
test_that("xportr_length: Accepts valid domain names in metadata object", {
```
Trying to keep generic.  @elimillera what d you think

Reply:
Maybe just `metadata or metacore object`

Reply:
That sounds good to me. There are no `metacore` references anywhere else in the documentation